beetsplug/parentwork.py

```diff
@@ -62,7 +61,7 @@ def find_parentwork(work_id):
     return(work_info)
 
 
-def get_info(item, work_info, parent_composer, parent_composer_sort,
+def get_info(self, item, work_info, parent_composer, parent_composer_sort,
```

Comment:
this is a function you explicitely asked me to put out of the class in your changes requested on the 11. of July. [EDIT] but you did it because there was no `self` in it... Ok, will change!